css/filmstrip/_small_video.scss

```diff
@@ -1,5 +1,5 @@
 .filmstrip__videos .videocontainer {
-    display: none;
+    display: inline-block;
```

Comment:
Now when you are asking I guess we didn't want to display it until we set the size for first time. Because during the resize we were also setting `display: inline-block`. Will test it out, think about it and if needed will revert.

Reply:
I can't see any side effects from this change. Lets keep it as it is now and if we receive reports about issues related to it, I'll revert the change. WDYT?

Reply:
Actually now we are setting the sizes when the videocontainer is created. That's why we shouldn't need this `display: none` -> `display:inline-block` transition.